src/main/java/org/forgerock/openam/auth/nodes/EmailNotifyNode.java

```diff
@@ -66,18 +68,19 @@ public interface Config {
         default String smtpHostName() { return "localhost"; }
```

Comment:
The changes introduced in the Config interface have an impact on the service config schema for this node. Have you tested if these changes are correctly upgraded when replacing the node with the newer version? If you don't want to deal with upgrade, then perhaps a note should be added to the readme at least?

Reply:
Yes, the plugin version change seems to trigger the service to upgrade the node gracefully.  The existing values are preserved, but the UI allows you to blank them out now if necessary.